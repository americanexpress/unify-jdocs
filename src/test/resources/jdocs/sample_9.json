{
  "docs": [
    {
      "index": 0,
      "name": "abc"
    }
  ]
}
