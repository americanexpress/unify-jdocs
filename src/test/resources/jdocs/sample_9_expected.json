{
  "docs": [
    {
      "index": 0,
      "name": "abc"
    },
    {
      "index": 1,
      "name": "xyz"
    }
  ]
}
